api/dependencies.py

```diff
@@ -11,23 +12,38 @@ def get_datetime_range(datetime_string: str | None) -> Tuple[Timestamp, Timestam
     if not datetime_string:
         return None
 
+    errors = {}
+
```

Comment:
As there are only two errors types, it is maybe more straightforward to just throw the exception directly?

Reply:
My initial thought was to make it expendable. So if we want to apply some other restrictions to time range in the future, we can respond with multiple invalid parameter messages at once and the user can correct all the mistakes at once instead of several requests.